docs/3.1. Discovery - Registered Operation.md

```diff
@@ -4,7 +4,35 @@ _(c) AMWA 2016, CC Attribution-ShareAlike 4.0 International (CC BY-SA 4.0)_
 
 This document describes usage of NMOS APIs for discovery in cases where where a distributed registry is available.
 
-## Registration
+## Registration API
+
+### DNS-SD Advertisement
+
+The preferred method of Registration API advertisement is via unicast DNS. Registration APIs MUST additionally be capable of producing an mDNS advertisement of the type \_nmos-register.\_tcp. This MAY be disabled via a user-configurable method.
```

Comment:
Suggest move service type from second to first sentence (and for other APIs):

The preferred method of Registration API advertisement is via a unicast DNS-SD advertisement of the type \_nmos-register.\_tcp. Registration APIs MUST additionally be capable of producing an mDNS advertisement. This MAY be disabled via a user-configurable method.